integration/acme_test.go

```diff
@@ -3,18 +3,22 @@ package main
 import (
 	"crypto/tls"
 	"errors"
-	"github.com/containous/traefik/integration/utils"
-	"github.com/go-check/check"
-	checker "github.com/vdemeester/shakers"
 	"net/http"
 	"os"
 	"os/exec"
 	"time"
+
+	"github.com/containous/traefik/integration/utils"
+	"github.com/go-check/check"
+	checker "github.com/vdemeester/shakers"
 )
 
 // ACME test suites (using libcompose)
 type AcmeSuite struct {
 	BaseSuite
+	onDemand            bool
```

Comment:
The others tests use one struct for BaseSuite and all the arguments they need.
I used only one to be homogeneous, but I can changed it if it's really necessary.